docs-chef-io/content/inspec/cli.md

```diff
@@ -1,17 +1,10 @@
 +++
-title = "InSpec CLI"
-draft = false
-gh_repo = "inspec"
-
-[menu]
-  [menu.inspec]
-    title = "InSpec Executable"
-    identifier = "inspec/reference/cli.md InSpec Executable"
-    parent = "inspec/reference"
-    weight = 10
+title: "InSpec CLI"
+draft: false
+gh_repo: "inspec"
 +++
 
-<!-- markdownlint-disable MD024 -->
```

Comment:
We should keep this if possible. See [MD024](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md024---multiple-headings-with-the-same-content)